core/src/main/scala/org/apache/spark/metrics/sink/Slf4jSink.scala

```diff
@@ -25,6 +25,9 @@ import com.codahale.metrics.{MetricRegistry, Slf4jReporter}
 import org.apache.spark.SecurityManager
 import org.apache.spark.metrics.MetricsSystem
 
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
```

Comment:
(It would be great if you run `sbt scalastyle` before adding some more commits which shows up such things)